news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What would you call your favorite tablet and why?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+Honestly I haven't used many tablets during my career since I always like to get something that's good quality and will last me a long time. My first tablet ever was the CTH-670 by Wacom, but sometime in 2021 I decided to get a CTH-680 since I hover and the hover distance was a lot better. I also really like the Wacom customization for the pen functions since I do art, so I think I will be sticking with the CTH-680 for a long time.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+How did you get introduced to osu! and what was your first impression of it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+My first introduction to osu! was actually in grade 4. I have a vivid memory of someone playing [Osu! Tatakae! Ouendan](/wiki/FAQ#i've-never-played-this-"ouendan"-thing-—-what-is-it) on their DS outside the gym, and I think there were some visuals of a guy trying to impress a girl on a date, and if you missed any notes then the guy would turn into a werewolf...
+
+Eventually when I started highschool, I saw people playing osu! on the school computers. It felt like some spiritual sign that I should play it since I always had that fond memory of that time by the gym. So at some point I tried it at home and the rest is prety much history. I don't remember much of my first days of the game but I knew I was hooked right away. I still play it to this day after all!
```

Comment:
```suggestion
Eventually when I started highschool, I saw people playing osu! on the school computers. It felt like some spiritual sign that I should play it since I always had that fond memory of that time by the gym. So at some point I tried it at home and the rest is pretty much history. I don't remember much of my first days of the game but I knew I was hooked right away. I still play it to this day after all!
```